microcosm_flask/conventions/logging_level.py

```diff
@@ -0,0 +1,103 @@
+"""
```

Comment:
Nope, that'll come; just wanted to see if anyone objected to the structure first before I entrenched it with tests.